src/lib.rs

```diff
@@ -144,26 +144,19 @@
 //! }
 //! ```
 //!
-//! # Features
-//!
-//! * `mio-support`: The [`Signals` iterator](iterator/struct.Signals.html) becomes pluggable into
-//!   mio 0.6.
-//! * `mio-0_7-support`: The [`Signals` iterator](iterator/struct.Signals.html) becomes pluggable into
-//!   mio 0.7.
-//! * `tokio-support`: The [`Signals`](iterator/struct.Signals.html) can be turned into
-//!   [`Async`](iterator/struct.Async.html), which provides a `Stream` interface for integration in
-//!   the asynchronous world.
+//! # Asynchronous runtime support
+//!
+//! If you are looking for integration with an asynchronous runtime take a look at one of the
+//! following adapter crates:
+//!
+//! * [`signal-hook-mio`](https://docs.rs/signal-hook-mio/latest) for mio support
+//! * [`signal-hook-mio-0_6`](https://docs.rs/signal-hook-mio-0_6/latest) for mio 0.6 support
```

Comment:
Nitpick/tip: Any reason to have the explicit `latest` at the end? I think the redirect adds it there once you go to the "base" URL, but it works all fine as `https://docs.rs/cratename`.

Reply:
Thanks for the tip :+1: